CHANGELOG.md

```diff
@@ -32,6 +32,9 @@
   interface MAC address incorrectly as `null`, and post snapshot restore as
   `00:00:00:00:00:00`. It now correctly reports the code generated MAC address
   when queried in both cases.
+- Fixed a self-DoS scenario in the virtio-queue code by reporting and
+  terminating execution when the number of available descriptors reported
```

Comment:
Does "reported by the driver" mean driver communicating to the device? Would it be better to choose another verb  to avoid confusion, e.g., "offered by the driver" as in the Virtio spec?

Reply:
yes, maybe 'offered by' would have been a better phrased. However, I already made the patch releases with exactly this changelog entry and I would keep it this way for consistency. Is this ok?

Reply:
Ok, then it's probably better to avoid divergence.